### PR TITLE
auto publish to npm

### PR DESCRIPTION
**Pull request by zhmushan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +35,9 @@
     "@prisma/client": "^2.1.0",
     "casbin": "^5.0.4"
   },
+  "files": [
+    "lib"
```
Comment (nodece) [inline package.json:39]: please append test、examples and prisma to files field.

Comment (zhmushan) [reply package.json:39]: These files are useless to users

Comment (hsluoyz) [reply package.json:39]: @nodece

Comment (nodece) [reply package.json:39]: It can help to users learn how use the prisma-adapter.

**Pull request merged**